MvvmCross/Platforms/Android/Views/MvxSplashScreenActivity.cs

```diff
@@ -88,9 +88,16 @@ public virtual async Task InitializationComplete()
 
         protected virtual async Task RunAppStartAsync(Bundle bundle)
         {
-            var startup = Mvx.IoCProvider.Resolve<IMvxAppStart>();
-            if (!startup.IsStarted)
-                await startup.StartAsync(GetAppStartHint(bundle));
+            if (Mvx.IoCProvider.TryResolve(out IMvxAppStart startup))
+            {
+                if (!startup.IsStarted)
```

Comment:
If `IsStarted` is true, shouldn't we call `Finish()` on the Splash Screen? There is no reason to keep it open?

Reply:
In my case I call `finish ()`.

Reply:
True! I'll make that change, also for the AppCompat version